clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -0,0 +1,2 @@
+// RUN: clang-doc --format=html --executor=standalone %s -output=%t/docs | FileCheck %s
+// CHECK: Using default asset: {{.*}}%{pathsep}share%{pathsep}clang
```

Comment:
The regular expression doesn't look right to me. I assume that `pathsep` will match eagerly after the `.*`. could be something else, though. I'm guessing you probably still need the `..`.

An alternate way to write the check is like this
```suggestion
// CHECK: Using default asset: 
// CHECK-SAME: %{pathsep}share%{pathsep}clang
```

Reply:
hmm, maybe I've sent you down the wrong path. I don't know if the `%{pathsep}` substitution will work outside of the `RUN` lines in something like FileCheck.

 If you cant get the old regex to pass, I'm fine w/ that.